lib/fog/aws/parsers/compute/describe_reserved_instances.rb

```diff
@@ -4,16 +4,15 @@ module Compute
       module AWS
         class DescribeReservedInstances < Fog::Parsers::Base
           def get_default_item
-            {'tagSet' => {}, 'recurringCharges' => {}}
+            {'tagSet' => {}, 'recurringCharges' => []}
           end
 
           def reset
             @context = []
-            # Note:  <recurringCharges> should also be handled as a set, but do not want to disrupt anyone relying on
-            # it currently being flatted
```

Comment:
I'm pretty sure this is false, nobody can be relying on this because it never worked :)